plugins/modules/hpilo_boot.py

```diff
@@ -76,6 +76,11 @@
     default: TLSv1
     type: str
     choices: ["SSLv3", "SSLv23", "TLSv1", "TLSv1_1", "TLSv1_2"]
+  idempotent_boot_once:
+    description:
+      - "This option makes O(state=boot_once) succeed instead of failing when the server is already powered on."
+    type: bool
+    default: false
```

Comment:
Sorry, it looks like we totally forgot to mention this:
```suggestion
    default: false
    version_added: 10.6.0
```
With that I think everything looks good!

Reply:
Thanks for the review ! 👍